packages/calcite-components/src/components/rating/rating.scss

```diff
@@ -84,13 +89,14 @@ calcite-chip {
 
 .number--average {
   font-weight: bold;
+  color: var(--calcite-rating-average-text-color);
```

Comment:
what is the purpose of this token?

Reply:
It sets the average text color, I set it to purple in this example. Should I remove this token?

![image](https://github.com/user-attachments/assets/81881fda-789e-40a4-a615-9f3856b2bf76)